fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -361,7 +361,11 @@ protected void handleQuery() {
                     ctx.setQueryId(UUIDUtil.genUUID());
                 }
                 parsedStmt = stmts.get(i);
-                //JDBC has no prepared prefix, only
+                // from jdbc no params like that. COM_STMT_PREPARE + select 1
+                if (ctx.getCommand() == MysqlCommand.COM_STMT_PREPARE && !(parsedStmt instanceof PrepareStmt)) {
+                    parsedStmt = new PrepareStmt("", parsedStmt, new ArrayList<>());
+                }
+                //only for JDBC
```

Comment:
this COM_STMT_PREPARE is not specific for jdbc, it's a binary protocol, changed to
// only for COM_STMT_PREPARE mode ?
// https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_com_stmt_prepare.html

Reply:
yes，but COM_STMT_PREPARE only happens in JDBC mode， mysql client only use plain text